package.json

```diff
@@ -30,8 +31,8 @@
   "scripts": {
     "test": "jest",
     "build": "lerna run build",
-    "lint:format": "tslint -p tsconfig.json --fix",
-    "lint": "tslint -p tsconfig.json",
+    "lint:format": "eslint --fix packages/ --ext .js,.ts",
```

Comment:
Is there any way to set it finely? I mean, I need to exclude the `build` path for each project. I usually test by build the lib before publishing, to check if they can be used in non TS environment.

Reply:
You can use [.eslintignore](https://eslint.org/docs/user-guide/configuring#eslintignore)

Reply:
Got it. I think I'll configure that by my self, thank you!